pkg/skaffold/generate_pipeline/constants.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generatepipeline
+
+const (
+	defaultKanikoSecretName = "kaniko-secret"
```

Comment:
can we just rename this to `kanikoSecretName` since "default" suggest there is way to override. Which i don't think it is, right?